controllers/base_controller.js

```diff
@@ -151,8 +154,26 @@ module.exports = function BaseControllerModule(pb) {
             onlyThisSite: true,
             siteObj: this.siteObj
         };
+        if(pb.config.localization && pb.config.localization.db){
```

Comment:
Making a call to the DB for every request to load localizations doesn't seem optimal.  Did you consider having a job to sync up the site specific options across the cluster in memory?